skel/go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/foolin/goview v0.3.0
 	github.com/frankban/quicktest v1.11.3 // indirect
 	github.com/gin-gonic/gin v1.7.2
+	github.com/golang-jwt/jwt v3.2.2+incompatible // indirect
```

Comment:
Is there a way to just use github.com/dgrijalva/jwt-go?
Try running `go mod tidy`